meshroom/ui/graph.py

```diff
@@ -118,8 +142,16 @@ def stop(self):
 
     @property
     def statusFiles(self):
```

Comment:
Need a dedicated property named for instance "watchedStatusFiles", as the statusFiles is semantically supposed to always return the list of all status files.

Reply:
Done